packages/migrate/src/helpers/transform-utils.ts

```diff
@@ -71,6 +71,21 @@ export function getTypeNameFromType(
   return typeName;
 }
 
+export function getTypeNameFromVariable(node: ts.Node, program: ts.Program): string | undefined {
+  const checker = program.getTypeChecker();
+  const type = checker.getTypeAtLocation(node);
+  if (!type) {
+    return undefined;
+  }
+  if (type.isNumberLiteral()) {
+    return 'number';
+  }
+  if (type.isStringLiteral()) {
+    return 'string';
```

Comment:
Just wondering if it make sense to have enum for types (for future fixes)?

Reply:
sounds good